e3sm_diags/derivations/acme.py

```diff
@@ -133,6 +133,16 @@ def qflxconvert_units(var):
     return var
 
 
+def w_convert_q(var):
+    if var.units == "mol/mol":
+        var = (
+            var * 18.0 / 28.97 * 1000.0
```

Comment:
I have updated the formula and comments. Thank you for the suggestion and review, @tangq